packages/api.md

```diff
@@ -10,3 +10,4 @@ It is composed of the following packages:
 | [@marlowe.io/runtime-rest-client](./modules/_marlowe_io_runtime_rest_client.html) | 1 to 1 mapping of the [Runtime Rest API](https://docs.marlowe.iohk.io/api/introduction).                                                                                                 |
 | [@marlowe.io/runtime-lifecycle](./modules/_marlowe_io_runtime_lifecycle.html)     | High level API that captures the life of "your" contracts. It builds upon the `wallet` and `runtime-rest-client` to solve common use cases                                               |
 | [@marlowe.io/adapter](./modules/_marlowe_io_adapter.html)                         | Supporting set of libraries for Marlowe and Runtime Core Domains.                                                                                                                        |
+| [@marlowe.io/marlowe-object](./modules/_marlowe_io_marlowe_object.html)           | Provides a way to define large contracts.                                                                                                                                                |
```

Comment:
`Specific contract model to optimize the definition of large contracts`

Reply:
more than optimize to allow/enable. Optimize implies that there is a way and this way is better. The only way we provide merkelization is through the `marlowe-object`. 

How about:

`An extension of the marlowe-core-v1 types that enable the creation of large contracts`?

Reply:
yeah better that the initial definition

Reply:
Will modify this as part of next PR